scripts/apps/contacts/components/ContactInfo.tsx

```diff
@@ -9,51 +9,54 @@ import {ContentDivider, Icon, Spacer} from 'superdesk-ui-framework/react';
 interface IProps {
     item: IContact;
     labelInactive?: string;
+    isActive?: boolean;
```

Comment:
what is active? contact info? What does it mean for contact info to be active. Better naming would help here. That labelInactive I'd rename too